### PR TITLE
Makes FCFlightLimitationViewController.swift more swifty.

### PR DESCRIPTION
**Pull request by valeriyvan**

### CODE REVIEW

File: Sample Code/SwiftSampleCode/DJISDKSwiftDemo/Components/FlightController/FlightLimitation/FCFlightLimitationViewController.swift [hunk 1]
```diff
@@ -17,131 +17,115 @@ class FCFlightLimitationViewController: DJIBaseViewController, DJIFlightControll
...
-            })
+                self?.showAlertResult("Get Max Flight Height:\(error.localizedDescription)")
+            }
             
-            fc?.flightLimitation?.getMaxFlightRadiusLimitationEnabledWithCompletion({[weak self](enabled: Bool, error: NSError?) -> Void in
+            fc.flightLimitation?.getMaxFlightRadiusLimitationEnabledWithCompletion { [weak self] (enabled: Bool, error: NSError?) -> Void in
                 
-                if error != nil {
-                    self?.showAlertResult("Get RadiusLimitationEnable:\(error?.localizedDescription)")
+                if let error = error {
```
Comment (athiercelin) [inline Sample Code/SwiftSampleCode/DJISDKSwiftDemo/Components/FlightController/FlightLimitation/FCFlightLimitationViewController.swift]: Same comment as before.

File: Sample Code/SwiftSampleCode/DJISDKSwiftDemo/Components/FlightController/LandingGear/FCLandingGearViewController.swift [hunk 2]
```diff
@@ -18,146 +18,97 @@ class FCLandingGearViewController: DJIBaseViewController, DJIFlightControllerDel
...
-                })
-            }
-            else {
+        if let fc = self.fetchFlightController() {
+            guard let landingGear = fc.landingGear else {
                 self.showAlertResult("Component Not Exist.")
+                return
+            }
+            landingGear.turnOffAutoLandingGearWithCompletion { [weak self] (error: NSError?) -> Void in
+                guard let error = error else { return }
```
Comment (athiercelin) [inline Sample Code/SwiftSampleCode/DJISDKSwiftDemo/Components/FlightController/LandingGear/FCLandingGearViewController.swift]: Same here too.

File: Sample Code/SwiftSampleCode/DJISDKSwiftDemo/Components/FlightController/LandingGear/FCLandingGearViewController.swift [hunk 3]
```diff
@@ -18,146 +18,97 @@ class FCLandingGearViewController: DJIBaseViewController, DJIFlightControllerDel
...
+                self?.showAlertResult("Turn Off:\(error.localizedDescription)")
             }
         }
     }
     
     @IBAction func onEnterTransportButtonClicked(sender: AnyObject) {
-        let fc: DJIFlightController? = self.fetchFlightController()
+        guard let fc = self.fetchFlightController() else { return }
```
Comment (athiercelin) [inline Sample Code/SwiftSampleCode/DJISDKSwiftDemo/Components/FlightController/LandingGear/FCLandingGearViewController.swift]: One liner.

File: Sample Code/SwiftSampleCode/DJISDKSwiftDemo/Components/FlightController/LandingGear/FCLandingGearViewController.swift [hunk 4]
```diff
@@ -18,146 +18,97 @@ class FCLandingGearViewController: DJIBaseViewController, DJIFlightControllerDel
...
+            self?.showAlertResult("Enter Transport: \(error.localizedDescription)")
         }
     }
     
     @IBAction func onExitTransportButtonClicked(sender: AnyObject) {
-        let fc: DJIFlightController? = self.fetchFlightController()
-        if (fc == nil) {
-            return
-        }
+        guard let fc = self.fetchFlightController() else { return }
```
Comment (athiercelin) [inline Sample Code/SwiftSampleCode/DJISDKSwiftDemo/Components/FlightController/LandingGear/FCLandingGearViewController.swift]: One liner.

File: Sample Code/SwiftSampleCode/DJISDKSwiftDemo/Components/FlightController/LandingGear/FCLandingGearViewController.swift [hunk 5]
```diff
@@ -18,146 +18,97 @@ class FCLandingGearViewController: DJIBaseViewController, DJIFlightControllerDel
...
-                }
-            })
-        }
-        else {
+        guard let landingGear = fc.landingGear else {
             self.showAlertResult("Component Not Exist.")
+            return
+        }
+        landingGear.exitTransportModeWithCompletion { [weak self] (error: NSError?) -> Void in
+            guard let error = error else { return }
```
Comment (athiercelin) [inline Sample Code/SwiftSampleCode/DJISDKSwiftDemo/Components/FlightController/LandingGear/FCLandingGearViewController.swift]: One liner

Comment (athiercelin) [inline Sample Code/SwiftSampleCode/DJISDKSwiftDemo/Components/FlightController/LandingGear/FCLandingGearViewController.swift]: Same here.

File: Sample Code/SwiftSampleCode/DJISDKSwiftDemo/Components/FlightController/LandingGear/FCLandingGearViewController.swift [hunk 6]
```diff
@@ -18,146 +18,97 @@ class FCLandingGearViewController: DJIBaseViewController, DJIFlightControllerDel
...
+            self?.showAlertResult("Exit Transport:\(error.localizedDescription)")
         }
     }
     
     func flightController(fc: DJIFlightController, didUpdateSystemState state: DJIFlightControllerCurrentState) {
-        let landingGear: DJILandingGear? = fc.landingGear
+        guard let landingGear = fc.landingGear else { return }
```
Comment (athiercelin) [inline Sample Code/SwiftSampleCode/DJISDKSwiftDemo/Components/FlightController/LandingGear/FCLandingGearViewController.swift]: One liner.

**Comment by oliverou** [comment]
Hi! Thanks for your contribution, we are actually working on refactoring the code for next release, will close this pull request.

**Pull request closed without merging**